# Coverage==5.0 fails with `Safety level may not be changed inside a transaction`
# on python 3.6.0 (xenial) (https://github.com/nedbat/coveragepy/issues/703)
coverage<5
jaraco.functools==4.1.0
jaraco.text==3.12.0
mock
more-itertools
mypy==1.14.1
platformdirs==4.3.6
pre-commit==1.21.0
pytest
tomli==2.2.1
types-PyYAML
# To resolve the error: types-requests 2.31.0.6 has requirement urllib3>=2, but you'll have urllib3 1.26.15 which is incompatible.
types-requests==2.31.0.5
